.changeset/odd-penguins-film.md

```diff
@@ -0,0 +1,10 @@
+---
+'@primer/react-brand': patch
+---
+
+Adds accessibility fixes.
```

Comment:
Suggestion for adding some clarity on which component this update will impact:

```suggestion
Added accessibility fixes for `Button`
```

Reply:
Good call! Should be adjusted now.